Package.swift

```diff
@@ -14,6 +15,10 @@ let package = Package(
     .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("c510e7d")),
     .package(url: "https://github.com/pointfreeco/swift-web.git", .revision("54412eb")),
     .package(url: "https://github.com/vapor/postgresql.git", from: "2.0.0"),
+
+    ProcessInfo.processInfo.environment["OSS"] == "1"
```

Comment:
we'll need to finesse the OSS experience a bit, but for now by default we'll use the private transcript repos.